src/index.html

```diff
@@ -4,9 +4,342 @@
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="MOYO logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+              data-qa="nav-hover">Laptops &#38; Computers </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="catalog main__catalog">
+        <div class="card catalog__card" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__image"
+          >
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+
+            <p class="card__review-value">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+
+          <a href="#" class="card__buy-btn"
+            data-qa="card-hover">
+            Buy
+          </a>
+        </div>
+
+        <div class="card catalog__card">
+          <img
+            src="images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+            class="card__image"
+          >
+
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="stars stars--4">
```

Comment:
the same as i mentioned higher about bem classes, and the same lower over all product cards